src/kernel/ship.c

```diff
@@ -172,7 +172,7 @@ struct ship *findshipr(const region * r, int n)
 void damage_ship(ship * sh, double percent)
 {
     double damage =
-        DAMAGE_SCALE * sh->type->damage * percent * sh->size + sh->damage;
+        DAMAGE_SCALE * sh->type->damage * percent * sh->size + sh->damage + .000001;
```

Comment:
Weil ich nicht will, dass in der nächsten Runde 1.9999999 auf 1 abgerundet wird. Man könnte auch richtig runden, aberr da Schaden positiv ist...